admin/src/pages/Creation.spec.js

```diff
@@ -93,121 +108,141 @@ describe('Creation', () => {
+            lastName: 'Bloxberg',
+            email: 'bibi@bloxberg.de',
+            creation: [200, 400, 600],
+            showDetails: false,
+          },
+        ])
+      })
 
-        it('removes the pushed item from itemsList', () => {
-          expect(wrapper.vm.itemsList).toEqual([
-            {
-              userId: 1,
-              firstName: 'Bibi',
-              lastName: 'Bloxberg',
-              email: 'bibi@bloxberg.de',
-              creation: [200, 400, 600],
-              showDetails: false,
-            },
-          ])
-        })
+      it('adds the pushed item to itemsMassCreation', () => {
+        expect(wrapper.vm.itemsMassCreation).toEqual([
+          {
+            userId: 2,
+            firstName: 'Benjamin',
+            lastName: 'Blümchen',
+            email: 'benjamin@bluemchen.de',
+            creation: [800, 600, 400],
+            showDetails: false,
+          },
+        ])
+      })
 
-        it('adds the pushed item to itemsMassCreation', () => {
-          expect(wrapper.vm.itemsMassCreation).toEqual([
-            {
-              userId: 2,
-              firstName: 'Benjamin',
-              lastName: 'Blümchen',
-              email: 'benjamin@bluemchen.de',
-              creation: [800, 600, 400],
-              showDetails: false,
-            },
-          ])
-        })
+      it('updates userSelectedInMassCreation in store', () => {
+        expect(storeCommitMock).toBeCalledWith('setUserSelectedInMassCreation', [
+          {
+            userId: 2,
+            firstName: 'Benjamin',
+            lastName: 'Blümchen',
+            email: 'benjamin@bluemchen.de',
+            creation: [800, 600, 400],
+            showDetails: false,
+          },
+        ])
+      })
+    })
 
-        describe('remove', () => {
-          beforeEach(() => {
-            wrapper.findComponent({ name: 'UserTable' }).vm.$emit(
-              'update-item',
-              {
-                userId: 2,
-                firstName: 'Benjamin',
-                lastName: 'Blümchen',
-                email: 'benjamin@bluemchen.de',
-                creation: [800, 600, 400],
-                showDetails: false,
-              },
-              'remove',
-            )
+    describe('remove item', () => {
+      beforeEach(async () => {
+        await wrapper.findComponent({ name: 'UserTable' }).vm.$emit('push-item', {
+          userId: 2,
+          firstName: 'Benjamin',
+          lastName: 'Blümchen',
+          email: 'benjamin@bluemchen.de',
+          creation: [800, 600, 400],
+          showDetails: false,
+        })
+        await wrapper
+          .findAllComponents({ name: 'UserTable' })
+          .at(1)
+          .vm.$emit('remove-item', {
+            userId: 2,
+            firstName: 'Benjamin',
+            lastName: 'Blümchen',
+            email: 'benjamin@bluemchen.de',
+            creation: [800, 600, 400],
+            showDetails: false,
           })
+      })
```

Comment:
`remove item` using `push-item` event - have I missunderstood something?

i think I do not really understand the test?! Is this the other list(source list) from which the emits stem from?